charts/osm/templates/osm-deployment.yaml

```diff
@@ -34,14 +34,18 @@ spec:
             "--sidecar-image", "{{.Values.OpenServiceMesh.sidecarImage}}",
             "--webhook-name", "osm-webhook-{{.Values.OpenServiceMesh.meshName}}",
             "--ca-bundle-secret-name", "{{.Values.OpenServiceMesh.caBundleSecretName}}",
-            "--certificate-manager", "{{.Values.OpenServiceMesh.certificateManager}}",
+            "--certificate-manager", "{{.Values.OpenServiceMesh.certficateManager}}",
+            {{- if .Values.OpenServiceMesh.enableVault}}
             "--vault-host", "{{.Values.OpenServiceMesh.vault.host}}",
             "--vault-protocol", "{{.Values.OpenServiceMesh.vault.protocol}}",
             "--vault-token", "{{.Values.OpenServiceMesh.vault.token}}",
-            "--cert-manager-issuer-name", "{{.Values.OpenServiceMesh.certmanager.issuerName}}",
-            "--cert-manager-issuer-kind", "{{.Values.OpenServiceMesh.certmanager.issuerKind}}",
-            "--cert-manager-issuer-group", "{{.Values.OpenServiceMesh.certmanager.issuerGroup}}",
+            {{- end }}
+            {{- if .Values.OpenServiceMesh.enableCertManager }}
+            "--certificate-manager-issuer-name", "{{.Values.OpenServiceMesh.certmanager.issuerName}}",
+            "--certificate-manager-issuer-kind", "{{.Values.OpenServiceMesh.certmanager.issuerKind}}",
+            "--certificate-manager-issuer-group", "{{.Values.OpenServiceMesh.certmanager.issuerGroup}}",
             "--service-cert-validity-minutes", "{{.Values.OpenServiceMesh.serviceCertValidityMinutes}}",
```

Comment:
`serviceCertValidityMinutes` should be outside the conditional as it is used by all of the certificate manager options

Reply:
thanks for spotting ths, fixed in last commit